io.github.enzomtpyt.pawtop.yml

```diff
@@ -0,0 +1,78 @@
+  - --filesystem=xdg-run/pipewire-0 # Pipewire interfacing
+  - --filesystem=xdg-run/speech-dispatcher # For TTS and VcNarrator
+  - --filesystem=~/.steam # Needed for SteamOS integration, as the XDG OpenURL portal doesn't work properly in Game Mode. We only need ~/.steam/steam.pipe but Flatpak can't correctly mount just that: 'File "/home/deck/.steam/steam.pipe" has unsupported type 0o10000'
+  - --env=XCURSOR_PATH=/run/host/user-share/icons:/run/host/share/icons # This is used to show the correct cursor on Wayland
+
+modules:
+  - name: Pawtop
+    buildsystem: simple
+    build-commands:
+      - chmod +x Pawtop.AppImage
+      - ./Pawtop.AppImage --appimage-extract
+      - desktop-file-edit --set-key="Exec" --set-value="startpawtop %U" --set-icon=$FLATPAK_ID
+        squashfs-root/pawtop.desktop
+      - |
+        for icon in 16 32 48 64 128 256 512; do
+          install -D squashfs-root/usr/share/icons/hicolor/${icon}x${icon}/apps/pawtop.png \
+              /app/share/icons/hicolor/${icon}x${icon}/apps/io.github.enzomtpyt.pawtop.png
+        done
+      - install -D squashfs-root/pawtop.desktop /app/share/applications/io.github.enzomtpyt.pawtop.desktop
+      - install -Dm755 startpawtop /app/bin/startpawtop
+      - install -D io.github.enzomtpyt.pawtop.metainfo.xml -t /app/share/metainfo/
+      - mv squashfs-root /app/bin/pawtop
+
+    sources:
+      - type: file
+        path: startpawtop
+
+      - type: file
+        url: https://raw.githubusercontent.com/enzomtpYT/Pawtop/main/meta/io.github.enzomtpyt.pawtop.metainfo.xml
+        sha512: de887ece72379323d101285f42763b00c082a3b24be5f6705c0b89e2c00b330a1bbd10a8b0b980e525634afdd9c7181c573f417b9f460a80919242bd2ba9600c
+        x-checker-data:
+          type: json
+          url: https://api.github.com/repos/enzomtpYT/Pawtop/releases/latest
+          version-query: .tag_name | sub("v"; "")
+          url-query: .assets[] | select(.name == "io.github.enzomtpyt.pawtop.metainfo.xml")
+            | .browser_download_url
+
+      - type: file
+        url: https://github.com/enzomtpYT/Pawtop/releases/latest/download/Pawtop-linux-x86_64.AppImage
+        dest-filename: Pawtop.AppImage
+        sha512: 68764a86be24540a98c80bbe42351f2d6082f29bd6ee5095597208ebd175c721b76ca43427c367c0a4cc2ee33ae29c81fb8bbedd30c1d3eda22ef8038e7da642
+        only-arches: [x86_64]
+        x-checker-data:
+          type: electron-updater
+          url: https://github.com/enzomtpYT/Pawtop/releases/latest/download/latest-linux.yml
+
+      - type: file
+        url: https://github.com/enzomtpYT/Pawtop/releases/latest/download/Pawtop-linux-arm64.AppImage
+        dest-filename: Pawtop.AppImage
+        sha512: fe87cd3693e84f1bb6853d027eabcbe201be501e187615e41d55c448e8462f48f08143781290b6a89ef57b65e0677979bf81cf581ca2e8639bb5fded9575b19e
+        only-arches: [aarch64]
+        x-checker-data:
+          type: electron-updater
```

Comment:
this is oss so it should be built from source https://docs.flathub.org/docs/for-app-authors/requirements#building-from-source